Set-Video-Mode/README.md

```diff
@@ -0,0 +1,105 @@
+# script-plugin-set-video-mode
+
+This repository contains a example of using Script Plugin and Plugin Message feature to set BrightSign VideoMode.
```

Comment:
** _an_ example

Reply:
Updated in [730000e](https://github.com/brightsign/BrightAuthor-Plugins/pull/20/commits/730000e99c8289a24d7a7f658c6f0f989cd53661)